runbook/net-runbook.sh

```diff
@@ -91,13 +97,21 @@ wrap-ipvsadm() {
 }
 
 wrap-net-eval() {
-  if [ "$MINOR_VERSION" -lt "11" ]; then
+  if [ "$MAJOR_VERSION" -lt 2 -a "$MINOR_VERSION" -lt "11" ]; then
     /opt/mesosphere/active/navstar/navstar/bin/navstar-env eval "$@"
   else
     /opt/mesosphere/bin/dcos-net-env eval "$@"
   fi
 }
 
+maybe-pprint-json() {
+  if hash jq 2> /dev/null; then
+    jq .
+  else
+    cat
+  fi
+}
+
```

Comment:
Could you split the changes into this PR into separate commits? Since many of the changes here are cleanup, it would be nice to at least separate the netstat changes, the conditional pretty printing, and other cleanup.